lib/matplotlib/sankey.py

```diff
@@ -599,34 +599,34 @@ def add(self, patchlabel='', flows=None, orientations=None, labels='',
-                  (Path.CURVE4, [(trunklength / 2.0 - self.gap) / 8.0,
-                                 -loss / 2.0]),
-                  (Path.LINETO, [(trunklength / 2.0 - self.gap) / 2.0,
-                                 -loss / 2.0]),
-                  (Path.LINETO, [(trunklength / 2.0 - self.gap),
-                                 -loss / 2.0])]
-        llpath = [(Path.LINETO, [(trunklength / 2.0 - self.gap),  # Lower left
-                                 loss / 2.0]),
-                  (Path.LINETO, [(trunklength / 2.0 - self.gap) / 2.0,
-                                 loss / 2.0]),
-                  (Path.CURVE4, [(trunklength / 2.0 - self.gap) / 8.0,
-                                 loss / 2.0]),
-                  (Path.CURVE4, [(self.gap - trunklength / 2.0) / 8.0,
-                                 -gain / 2.0]),
-                  (Path.LINETO, [(self.gap - trunklength / 2.0) / 2.0,
-                                 -gain / 2.0]),
-                  (Path.LINETO, [(self.gap - trunklength / 2.0),
-                                 -gain / 2.0])]
-        lrpath = [(Path.LINETO, [(trunklength / 2.0 - self.gap),  # Lower right
-                                 loss / 2.0])]
-        ulpath = [(Path.LINETO, [self.gap - trunklength / 2.0,  # Upper left
-                                 gain / 2.0])]
+        urpath = [(Path.MOVETO, [(self.gap - trunklength / 2),  # Upper right
+                                 gain / 2]),
+                  (Path.LINETO, [(self.gap - trunklength / 2) / 2,
+                                 gain / 2]),
+                  (Path.CURVE4, [(self.gap - trunklength / 2) / 8,
+                                 gain / 2]),
+                  (Path.CURVE4, [(trunklength / 2 - self.gap) / 8,
+                                 -loss / 2]),
+                  (Path.LINETO, [(trunklength / 2 - self.gap) / 2,
+                                 -loss / 2]),
+                  (Path.LINETO, [(trunklength / 2 - self.gap),
+                                 -loss / 2])]
+        llpath = [(Path.LINETO, [(trunklength / 2 - self.gap),  # Lower left
+                                 loss / 2]),
+                  (Path.LINETO, [(trunklength / 2 - self.gap) / 2,
+                                 loss / 2]),
+                  (Path.CURVE4, [(trunklength / 2 - self.gap) / 8,
+                                 loss / 2]),
+                  (Path.CURVE4, [(self.gap - trunklength / 2) / 8,
+                                 -gain / 2]),
+                  (Path.LINETO, [(self.gap - trunklength / 2) / 2,
+                                 -gain / 2]),
+                  (Path.LINETO, [(self.gap - trunklength / 2),
+                                 -gain / 2])]
+        lrpath = [(Path.LINETO, [(trunklength / 2 - self.gap),  # Lower right
+                                 loss / 2])]
+        ulpath = [(Path.LINETO, [self.gap - trunklength / 2,  # Upper left
+                                 gain / 2])]
```

Comment:
I'm fairly certain that all of these point lists could fit on a single line now, if you remove the comment and put it above.

Reply:
Oh yes, now it fits. Thank you for pointing out that (and for fixing my typo in the title). Fixed, repushed.